src/test/scala/mesosphere/marathon/core/leadership/AlwaysElectedLeadershipModule.scala

```diff
@@ -0,0 +1,17 @@
+package mesosphere.marathon.core.leadership
+
+import akka.actor.{ ActorRefFactory, ActorRef, Props }
+import mesosphere.marathon.core.base.{ ActorsModule, ShutdownHooks }
+
+private class AlwaysElectedLeadershipModule(actorRefFactory: ActorRefFactory) extends LeadershipModule(actorRefFactory) {
```

Comment:
Please add some documentation for this module as well the provided objects/functionality.